qutebrowser/commands/userscripts.py

```diff
@@ -315,6 +315,29 @@ def prepare_run(self, *args, **kwargs):
             return
 
 
+class NotFoundError(Exception):
+
+    """Raised when spawning a userscript that doesn't exist.
+
+    Attributes:
+        script_name: name of the userscript as called
+        paths: path names that were searched for the userscript
+        message: descriptive message string of the error
```

Comment:
My (possibly bad) habit of caching... I'll move it to `str()` in combination with the string formatting